js/runtime.js

```diff
@@ -1,9 +1,11 @@
 (function(global){
     'use strict';
 
-    // Dependencies: ctx, canvas, Event, runtime, sound, soundEffect,
+    // Dependencies: ctx, canvas, Event, runtime
     // canvas/stage stuff
     var _canvas, _ctx;
+    var song = "o4 l4 V12 ";
```

Comment:
Yeah just need to set it up with those parameters specifying the octave, default volume, and volume before notes begin being added.